### DISCUSSION TITLE
Can I use this site as a template for one I am working on

### DISCUSSION THREAD

**Discussion by leahcimto** [theforumhelpers/theforumhelpers.github.io#233] [Q&A] [answered]
I'm wondering if it is allowed I use code in this site as a template for one I am working on. I would credit this site of course and change all the content but the layout.

**Comment by Accio1** [comment, marked as answer]
Yes, since the site is open source, you can use the code.

**Comment by BarelySmooth** [comment]
Also, see the license file of the project which gives you a greater idea of how you can use the code!
